src/main/app/src/app/klanten/persoonsgegevens/persoonsgegevens.component.html

```diff
@@ -30,7 +31,7 @@
           mat-icon-button
           [routerLink]="['/persoon', persoon.bsn]"
           (click)="$event.stopPropagation()"
-          [id]="'persoonBekijken_' + persoon.bsn +'_button'"
+          [id]="'persoonBekijken_' + persoon.bsn + '_button'"
```

Comment:
Ha, tour VSC default setting also has this pref 😉

Reply:
![breath](https://media.tenor.com/-QBEVIuCYWYAAAAC/breathe-breathe-deep.gif)